src/app/backend/dashboard.go

```diff
@@ -76,14 +77,17 @@ func main() {
 
 	log.Printf("Successful initial request to the apiserver, version: %s", versionInfo.String())
 
-	heapsterRESTClient, err := heapster.CreateHeapsterRESTClient(*argHeapsterHost,
-		apiserverClient)
+	// Init integrations
+	integrationManager := integration.NewIntegrationManager(clientManager)
+	err = integrationManager.Metric().
+		ConfigureHeapster(*argHeapsterHost).
+		Enable(integrationapi.HeapsterIntegrationID)
```

Comment:
Any integration still requires some code to be pushed upstream. We'd need to implement prometheus client that implements `MetricClient` interface. 

Normally with settings page user will be able to refresh state of integrations, i.e. if prometheus is not installed in the cluster then user goes to settings page clicks install near prometheus integration and then enables it. That would automatically disable other metric clients.